packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -39,6 +39,11 @@ import {slugify} from '../utility'
 import {useAuthorListQuery, AuthorRefFragment, ImageRefFragment} from '../api'
 
 import {useTranslation} from 'react-i18next'
+export interface ArticleMetadataProperty {
```

Comment:
why did you add this?

Reply:
Same here

Reply:
Sorry. I got confused. I didn't know that github highlighted those changes.